restaurant2osm.py

```diff
@@ -523,7 +529,7 @@ def geocode (street, house_number, house_letter, city):
 				make_osm_line("LAST_INSPECTION", restaurant['date_last_inspection'])
 
 				if restaurant['name'] != restaurant['original_name']:
-					make_osm_line("ORIGINAL_NAME", restaurant['original_name'])
+					make_osm_line("official_name", restaurant['original_name'])
```

Comment:
Til å bruke som en slags referanse for matching

Reply:
Tror ikke kvaliteten er god nok til å ha med i OSM, dessuten har nok ikke Mattilsynet angitt det offisielle navnet

Reply:
Fjernet denne endringen